examples/src/main/scala/src/main/scala/za/co/absa/enceladus/examples/CustomRuleSample4.scala

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package src.main.scala.za.co.absa.enceladus.examples
+
+import java.io.File
+
+import org.apache.spark.sql.{Column, DataFrame, DataFrameReader, SparkSession}
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.functions.col
+import src.main.scala.za.co.absa.enceladus.examples.interpreter.rules.custom.{LPadCustomConformanceRule, UppercaseCustomConformanceRule}
+import za.co.absa.enceladus.conformance.CmdConfig
+import za.co.absa.enceladus.conformance.interpreter.DynamicInterpreter
+import za.co.absa.enceladus.dao.{EnceladusDAO, EnceladusRestDAO}
+import za.co.absa.enceladus.model.Dataset
+import org.apache.log4j.{Level, Logger}
+import scopt.OptionParser
+
+object CustomRuleSample4 {
+  /**
+    * This is a class for configuration provided by the command line parameters
+    *
+    * Note: scopt requires all fields to have default values.
+    *       Even if a field is mandatory it needs a default value.
+    */
+  private case class CmdConfigLocal(inputFormat: String = "csv",
+                            rowTag: Option[String] = None,
```

Comment:
Copy&paste artifact. Moved as `CmdConfig` class, caused conflict with the `implicit`, renamed to `CmdConfigLocal` which pushed the first line parenthesis not the rest  (this is why I  sometimes use on multi-line declaration one indentation instead of end-of-name alignment.
Anyway, thanks for nothing, adjusted.